src/openrct2/world/TileInspector.cpp

```diff
@@ -35,6 +35,7 @@ uint32_t windowTileInspectorTileX;
 uint32_t windowTileInspectorTileY;
 int32_t windowTileInspectorElementCount = 0;
 int32_t windowTileInspectorSelectedIndex;
+uint8_t windowTileInspectorTrackSpeed;
```

Comment:
Get rid of this, there is no need to store this value anywhere, this data already exists in the element itself. This just introduces code where you need to set it (currently on selecting a tile), making it quite phrone to errors (e.g. when refactoring or introducing a new way of selecting tiles for instance).